assets/js/OutputPane.js

```diff
@@ -158,7 +160,32 @@ export default class OutputPane extends React.Component {
       });
   }
 
+  findCurrentModuleInWorkflow(wf) {
+      if(!wf) {
+          return null;
+      }
+      if(!wf.wf_modules) {
+          return null;
+      }
+      var modulesFound = wf.wf_modules.filter((wfm) => {return wfm.id == this.props.selectedWfModuleId});
+      return modulesFound.length > 0 ? modulesFound[0] : null;
+  }
+
   render() {
+    // We figure out whether we need to indicate sort status here so that we don't have to
+    // pass a ton of data to the TableView
```

Comment:
I'm not super happy putting all this sort code in here, it really belongs in TableView. Can we just pass the currentModule to tableView and let it figure out the sort thing there?